src/includes/content/wizard/affliate/flycricket.pug

```diff
@@ -4,13 +4,12 @@
     #fc-callouts
         form#fc-form(action='https://app.flycricket.com/import/privacy' method='post' target='_blank' onsubmit='return fc_submit(event)')
             input(type='hidden' name='fc-referrer' value='nishant')
+            input#fc-terms(type='hidden' name='fc-terms')
             input#fc-body(type='hidden' name='fc-body')
```

Comment:
maybe we can rename all the `fc-body` to `fc-privacy` to be consistent and more meaningful

Reply:
Good call - added a commit with this change!